### PR TITLE
Mn/marketing top nav

### PR DESCRIPTION
**Pull request by mckayn10**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,13 @@
 import React, { Component } from 'react';
 
 import './App.css';
+import MarketingLanding from './views/marketing/MarketingLanding';
```
Comment (huyqdang) [inline src/App.js]: this is being imported but never used

File: .gitignore [hunk 2]
```diff
@@ -3,6 +3,7 @@
 # dependencies
 /node_modules
 /.pnp
+/package-lock.json
```
Comment (huyqdang) [inline .gitignore]: also for some reasons the package lock is not being ignore.

Comment (mckayn10) [reply .gitignore]: I added "package-lock.json" to the .gitignore

File: src/App.js (hunk 1 above)
Comment (mckayn10) [reply src/App.js]: I deleted this import

**Review by huyqdang — APPROVED** [review]

**Pull request merged**